bin/test.sh

```
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+test() {
+  echo "Test $1"
+  cd $1
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)